openedx/features/job_board/urls.py

```diff
@@ -4,5 +4,5 @@
 
 urlpatterns = [
     url(r'^jobs/$', JobListView.as_view(), name='job_list'),
-    url(r'^jobs/[0-9]+$', show_job_detail, name='job_detail'),
+    url(r'^jobs/(?P<job_id>[0-9]+)$', show_job_detail, name='job_detail'),
```

Comment:
Please update `job_detail_page = reverse('job_detail')` in theme, because it is breaking code. Add job id as well to reverse at `src/philu-edx-theme/edx-platform/philu/lms/templates/features/job_board/job_list.html:9`

Reply:
Removing `src/philu-edx-theme/edx-platform/philu/lms/templates/features/job_board/job_list.html:9` and replacing line 124 with `<h4><a href="${ reverse('job_detail', args=[job.id]) }"> ${ job.title } </a></h4>` would work.